internal/ent/schema/origin.go

```diff
@@ -40,6 +40,11 @@ func (Origin) Fields() []ent.Field {
 			Annotations(
 				entgql.OrderField("name"),
 			),
+		field.Int("weight").
+			Default(100).
+			Annotations(
```

Comment:
should there be a `NotEmpty()` in there too? Or should users be able to set it to nothing?

Reply:
`NotEmpty` isn't available on `Int` fields. That's why I set the default to `100` if nothing is provided though.